backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/utils/CassandraAsyncExecutor.java

```diff
@@ -67,7 +68,7 @@ public CassandraAsyncExecutor(Session session) {
 
     public Flux<Row> executeRows(Statement statement) {
         return execute(statement)
-            .flatMapMany(Flux::fromIterable);
+            .flatMapIterable(Function.identity());
```

Comment:
Can you explain what `Function.identity()` is for exactly?

Reply:
We have a `Mono<ResultSet>` and `ResultSet` is an interable, so we do not need to transform it.